### PR TITLE
Turn keepa both async and sync

### PR DESCRIPTION
**Pull request by chocobn69**

### CODE REVIEW

**Comment by chocobn69** [comment]
Do you know how to fix this ? I think > 90% coverage is ok

**Comment by akaszynski** [comment]
@chocobn69, I've patched the code cov yml.  Good to go.  Is this ready for review?

**Comment by chocobn69** [comment]
I'm trying this code on my prod app today.

Documentation is missing

**Comment by akaszynski** [comment]
Documentation with some simple examples would be great.  Thanks for all your work on this!

**Comment by chocobn69** [comment]
Code is running fine since 3 days on my production app, ok for me

File: docs/source/index.rst [hunk 1]
```diff
@@ -39,6 +39,34 @@ Here's an example of obtaining a product and plotting its price and offer histor
 
     Product Offers Plot
 
+Brief Example (async)
+-------------
```
Comment (akaszynski) [inline docs/source/index.rst:43]: rst docstring fix:

```suggestion
Brief Example using async
-------------------------
```

File: keepa/_version.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 """Version number for keepa
 """
 # major, minor, patch, -extra
-version_info = 0, 20, 1
+version_info = 1, 0, 0
```
Comment (akaszynski) [inline keepa/_version.py:4]: Probably a good idea to finally get this to `1.0.0` since the API is basically stable at this point.

**Comment by akaszynski** [comment]
This is great!

Tests all pass locally as well, and I'm pleased with how much work you put in to make `async` work for the interface.

Well done!

**Pull request merged**